kim_cnn/README.md

```diff
@@ -35,24 +35,48 @@ python main.py --trained_model saves/best_model.pt --mode multichannel
 
 
 ## Dataset and Embeddings 
+- Dataset is in data repo. You can set your own path in *SST1.py* and *args.py* for data and embeddings.
```

Comment:
Add instructions to clone the data repo and how the directory structure looks like. Something like [this](https://github.com/castorini/Castor/blob/master/sm_modified_cnn/README.md).